### PR TITLE
Milestone 5 final one! PR

### PR DESCRIPTION
**Pull request by JieningTao**

### CODE REVIEW

File: Assets/Scripts/BlocadeScript.cs [hunk 1]
```diff
@@ -17,34 +17,54 @@ public class BlocadeScript : MonoBehaviour
...
+        blockScripts[4].delayTimer = 1f;
+    }
+
+    private void OnCrowCommand(List<string> commands)
+    {
+        if (commands[0] == "OpenBlocade")
+            OpenBlocade();
+    }
+
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/BlocadeScript.cs]: Whitespace

File: Assets/Scripts/BlocadeScript.cs [hunk 2]
```diff
@@ -17,34 +17,54 @@ public class BlocadeScript : MonoBehaviour
...
+    private void OpenBlocade()
+    {
+
+        blockScripts[2].moving = true;
+
+        blockScripts[1].moving = true;
+        blockScripts[1].delayTimer = 0.5f;
+        blockScripts[3].moving = true;
+        blockScripts[3].delayTimer = 0.5f;
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/BlocadeScript.cs:46]: Whitespace

File: Assets/Scripts/Credits.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Credits : MonoBehaviour
+{
+    public void RadarCred()
```
Comment (TheQuestCrafter) [inline Assets/Scripts/Credits.cs:8]: Add space before method

File: Assets/Scripts/DontDestroyOnLoad.cs [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DontDestroyOnLoad : MonoBehaviour
+{
+    void Awake()
```
Comment (TheQuestCrafter) [inline Assets/Scripts/DontDestroyOnLoad.cs:8]: Added space before  method

File: Assets/Scripts/DontDestroyOnLoad.cs [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+    {
+        GameObject[] objs = GameObject.FindGameObjectsWithTag("Inventory");
+
+        if (objs.Length > 1)
+        {
+            Destroy(this.gameObject);
+        }
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/DontDestroyOnLoad.cs]: Whitespace

File: Assets/Scripts/FollowObject.cs [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+
+    private void Start()
+    {
+        thisObjectScript = GetComponent<InventoryObject>();
+    }
+
+
+
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/FollowObject.cs]: Whitespace

File: Assets/Scripts/FollowObject.cs [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+public class FollowObject : MonoBehaviour
+{
+    [SerializeField]
+    private float Hoverheight;
+
+
+    private Transform PlayerLocation;
+    private InventoryObject thisObjectScript;
+
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/FollowObject.cs]: Whitespace

File: Assets/Scripts/FollowObject.cs [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+            
+            follow();
+        }
+    }
+
+    private void follow()
+    {
+        this.transform.position = PlayerLocation.position + new Vector3(0, Hoverheight, 0);
+    }
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/FollowObject.cs:30]: Whitespace

File: Assets/Scripts/InventoryObject.cs [hunk 9]
```diff
@@ -46,9 +56,32 @@ public override void InteractWith()
...
+                renderer.enabled = false;
+            else
+            {
+                foreach (Renderer r in childrenrenders)
+                {
+                    r.enabled = false;
+                }
+
+            }
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/InventoryObject.cs:75]: Whitespace

File: Assets/Scripts/RisingStairs.cs [hunk 10]
```diff
@@ -10,12 +10,21 @@ public class RisingStairs : MonoBehaviour
     private bool rising = false;
     [SerializeField]
     Transform stairs;
+    [SerializeField]
```
Comment (TheQuestCrafter) [inline Assets/Scripts/RisingStairs.cs:16]: Add space before serialized fields

File: Assets/Scripts/SequenceLights.cs [hunk 11]
```diff
@@ -4,24 +4,21 @@
...
-    [SerializeField]
-    private BoxCollider underPreviousLight;
-    [SerializeField]
-    private GameObject spotlight;
 
-    //this class is a work in progress, very early
 
-    // Start is called before the first frame update
+
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/SequenceLights.cs]: Whitespace

File: Assets/Scripts/SpinningScript.cs [hunk 12]
```diff
@@ -0,0 +1,13 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SpinningScript : MonoBehaviour
+{
+    [SerializeField]
+    private float spinSpeed;
+    private void FixedUpdate()
+    {
+       //transform.rotation = transform.rotation.y + spinSpeed;
```
Comment (TheQuestCrafter) [inline Assets/Scripts/SpinningScript.cs]: Delete unused method

File: Assets/Scripts/StuffToAddByCrow.cs [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+public class StuffToAddByCrow : MonoBehaviour
+{
+    // Start is called before the first frame update
+    [SerializeField]
+    private string commandToSpawn;
+
+    [SerializeField]
+    private GameObject objectToSpawn;
+
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/StuffToAddByCrow.cs]: Whitespace

File: Assets/Scripts/StuffToAddByCrow.cs [hunk 14]
```diff
@@ -0,0 +1,40 @@
...
+
+    private void OnCrowCommand(List<string> commands)
+    {
+        if (commands[0] == commandToSpawn)
+        {
+            objectToSpawn.SetActive(true);
+        }
+
```
Comment (TheQuestCrafter) [inline Assets/Scripts/StuffToAddByCrow.cs]: Whtiespace

**Review by TheQuestCrafter — CHANGES_REQUESTED** [review]

File: Assets/Scripts/BlocadeScript.cs (hunk 2 above)
Comment (JieningTao) [reply Assets/Scripts/BlocadeScript.cs:46]: intentionally left for better organization

File: Assets/Scripts/SpinningScript.cs (hunk 12 above)
Comment (JieningTao) [reply Assets/Scripts/SpinningScript.cs]: script deleted

**Pull request merged**